cli/src/install.sh

```diff
@@ -48,87 +53,97 @@ get_rc_file() {
-        success "Updated ${shell}'s \$PATH to include ${HOME}/.phylum/."
-    fi
-
-    if ! grep -q 'alias ph=' "${rc_path}"; then
-        echo "alias ph='phylum'" >> "${rc_path}"
-        success "Created ph alias for phylum in ${shell}."
-    fi
-}
-
 patch_zshrc() {
+    phylum_rc="${data_dir}/zshrc"
     rc_path="${HOME}/.zshrc"
 
     if [ ! -f "${rc_path}" ]; then
         touch "${rc_path}"
     fi
 
-    if ! grep -q '.phylum/completions' "${rc_path}"; then
-        echo "fpath+=(\"\$HOME/.phylum/completions\")" >> "${rc_path}"
-    fi
-    if ! grep -q 'autoload -U compinit && compinit' "${rc_path}"; then
-        echo "autoload -U compinit && compinit" >> "${rc_path}"
+    echo "\
+export PATH=\"${bin_dir}:\$PATH\"
+alias ph='phylum'
+fpath+=(\"${completions_dir}\")
+autoload -U compinit && compinit" \
+    > "${phylum_rc}"
+
+    if ! grep -q "source ${phylum_rc}" "${rc_path}"; then
+        echo "source ${phylum_rc}" >> "${rc_path}"
     fi
-    success "Completions are enabled for zsh."
 
-    add_to_path_and_alias "${rc_path}" "zsh"
+    success "Completions are enabled for zsh."
 }
 
 patch_bashrc() {
+    phylum_rc="${data_dir}/bashrc"
     rc_path="${HOME}/.bashrc"
 
     if [ ! -f "${rc_path}" ]; then
         touch "${rc_path}"
     fi
 
-    if ! grep -q '.phylum/completions/phylum.bash' "${rc_path}"; then
-        echo "source \$HOME/.phylum/completions/phylum.bash" >> "${rc_path}"
-    fi
-    success "Completions are enabled for bash."
+    echo "\
+export PATH=\"${bin_dir}:\$PATH\"
+alias ph='phylum'
+source ${completions_dir}/phylum.bash" \
+    > "${phylum_rc}"
 
-    add_to_path_and_alias "${rc_path}" "bash"
-}
+    if ! grep -q "source ${phylum_rc}" "${rc_path}"; then
+        echo "source ${phylum_rc}" >> "${rc_path}"
+    fi
 
-create_directory() {
-    # Create the config directory if one does not already exist.
-    install -d "${HOME}/.phylum"
+    success "Completions are enabled for bash."
 }
 
 copy_files() {
     # Copy the specific platform binary.
     platform=$(set -e; get_platform)
     bin_name="phylum"
 
-    install -m 0755 "${bin_name}" "${HOME}/.phylum/phylum"
+    # Ensure binary directory exists.
+    mkdir -p "${bin_dir}"
+
+    install -m 0755 "${bin_name}" "${bin_dir}/${bin_name}"
     if [ "${platform}" = "macos" ]; then
         # Clear all extended attributes. The important one to remove here is 'com.apple.quarantine'
         # but there might be others or there might be none. `xattr -c` works in all of those cases.
-        xattr -c "${HOME}/.phylum/phylum"
+        xattr -c "${bin_dir}/${bin_name}"
     fi
 
-    # Ensure correct permissions on settings.yaml (if it exists).
-    if [ -f "${HOME}/.phylum/settings.yaml" ]; then
-        chmod 600 "${HOME}/.phylum/settings.yaml"
+    # Correct inaccurate settings.yaml permissions set by previous installers.
+    if [ -f "${config_dir}/settings.yaml" ]; then
+        chmod 600 "${config_dir}/settings.yaml"
     fi
```

Comment:
The previous installer that we're fixing would have put this file in the old location, so this doesn't actually accomplish what it says.

So I think we've got three options:

1. Use the old location for this section of the install script
2. Correct the permissions in the migration code in `config.rs` instead of here
3. Declare that we're not worried about this anymore and delete these lines

I'm okay with any of the 3 options.

Reply:
> The previous installer that we're fixing would have put this file in the old location, so this doesn't actually accomplish what it says.

It does' but it assumes that the user ran `phylum update`. The configuration file is migrated by `phylum` sooner or later, but that doesn't do any permission changes.

We could also try to fix all permission changes in the move (in Rust) though, then it should be fixed once and forever and we can remove it from the installer?

Reply:
> It does' but it assumes that the user ran `phylum update`. The configuration file is migrated by `phylum` sooner or later, but that doesn't do any permission changes.

Good point... So if we want to fix this in `install.sh`, we would have to check both the old location (in case of a manual update where a user downloads the new zip and runs `install.sh`) and the old location (in case of `phylum update`)... That's annoying
 
> We could also try to fix all permission changes in the move (in Rust) though, then it should be fixed once and forever and we can remove it from the installer?

Yep. That would do it.